docs/devtools.md

```diff
@@ -87,3 +90,7 @@ Use these options to style the dev tools.
   - The standard React style object used to style a component with inline styles
 - `className: string`
   - The standard React className property used to style a component with classes
+- `showCloseButton?: boolean`
```

Comment:
I tried this prop in codesandbox, but it seems that it doesn't do anything?

https://codesandbox.io/s/tanstack-query-example-react-basic-forked-x3s37v?file=/src/index.jsx

In that sandbox, it is set to `false`, but the closeButton still shows up.

Reply:
I will fix that